addons/website_forum/controllers/website_forum.py

```diff
@@ -111,11 +113,12 @@ def _get_forum_post_search_options(self, forum=None, tag=None, filters=None, my=
                  '''/forum/<model("forum.forum"):forum>/tag/<model("forum.tag"):tag>/questions''',
                  '''/forum/<model("forum.forum"):forum>/tag/<model("forum.tag"):tag>/questions/page/<int:page>''',
                  ], type='http', auth="public", website=True, sitemap=sitemap_forum)
-    def questions(self, forum=None, tag=None, page=1, filters='all', my=None, sorting=None, search='', create_uid=False, include_answers=False, **post):
+    def questions(self, forum=None, tag=None, page=1, filters='all', my=None, sorting=None, search='', create_uid=False,
+                  include_answers=False, visible=True, **post):
         Post = request.env['forum.post']
 
         author = request.env['res.users'].browse(int(create_uid))
-
+        visible = visible not in {'false', 'False', False, 0}
```

Comment:
```
from odoo.tools.misc import str2bool
```

```suggestion
        visible = visible and str2bool(visible)
```


shoulodn't we do the same for `include_answers` ?